pkgs/top-level/php-packages.nix

```diff
@@ -312,7 +312,7 @@ lib.makeScope pkgs.newScope (self: with self; {
 
     phalcon = callPackage ../development/php-packages/phalcon { };
 
-    php-spx = callPackage ../development/php-packages/php-spx { };
+    php-spx = lib.warn "php-spx is deprecated, use spx instead" self.extensions.spx;
```

Comment:
Warnings are not generally allowed in packages since they spam evaluation. You should use `throw` and make it conditional on `config.allowAliases`.

Reply:
Thanks @jtojnar!

Reply:
Indeed, thanks !